GoogleChromeLabs/ps-analysis-tool
# Feature: Filter navigation from aggregated report

## Description
This PR enables navigating from a sitemap report insights page to the first site top frame when any filter is selected.


## Relevant Technical Choices
- As the category filter contains only 4 options, we will not calculate them from data, but keep them static in code.


## Testing Instructions
- Build CLI and analyze a sitemap.
- Open the HTML file and click on the Functional category on the sitemap report's landing page.
- It should open the first site's top frame with a functional filter selected.
- Now navigate to other sites' different frames, and the filter should persist.
- Try other filter selections, the navigation, and persistence should persist across the sites and frames.
- Now analyze a single site and check whether everything works correctly with the same scenarios.


## Additional Information:



## Screenshot/Screencast

https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/1a19f50e-772f-4cd0-849e-f00aa01eee89




---

## Checklist






Fixes #737
